app/src/main/java/com/example/hungdev/musicplayer/data/source/local/SongLocalDataSource.java

```diff
@@ -0,0 +1,61 @@
+package com.example.hungdev.musicplayer.data.source.local;
+
+import android.content.ContentResolver;
+import android.content.Context;
+import android.database.Cursor;
+import android.media.MediaMetadataRetriever;
+import android.net.Uri;
+import android.provider.MediaStore;
+import android.widget.Toast;
+
+import com.example.hungdev.musicplayer.data.Song;
+import com.example.hungdev.musicplayer.data.source.SongDataSource;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by hungdev on 06/08/2018.
+ */
+
+public class SongLocalDataSource implements SongDataSource{
+
+    private Context mContext;
+    public SongLocalDataSource(Context mContext) {
+        this.mContext = mContext;
+    }
+
+    @Override
+    public void getSongs(LoadSongsCallback callback) {
+        List<Song> mSongs = new ArrayList<>();
```

Comment:
rename to songs,
các biến instance variable thì mới khai báo ntn thui nhé,
các biến local khai báo camelCase